ui/apps/everest/.e2e/utils/table.ts

```diff
@@ -21,7 +21,7 @@ export const findRowAndClickActions = async (
 export const waitForInitializingState = async (page: Page, name: string) => {
   const dbRow = page.getByRole('row').filter({ hasText: name });
   await expect(dbRow).toBeVisible();
-  await expect(dbRow.getByText('Unknown')).not.toBeVisible({ timeout: 15000 });
+  await expect(dbRow.getByText('Creating')).not.toBeVisible({ timeout: 15000 });
```

Comment:
shouldn't this be just `toBeVisible`, without `not`?

Reply:
no, never mind. This is still OK, because we want to make sure the DB is not on the initial stage